pro/src/components/pages/Offerers/Offerer/VenueV1/fields/ReimbursementPoint/ReimbursementPoint.jsx

```diff
@@ -148,6 +149,12 @@ const ReimbursementPoint = ({
             {!venueReimbursementPoint && !!reimbursementPointOptions.length && (
               <p className={styles['or-separator']}>ou</p>
             )}
+            {isDialogOpen && (
+              <ReimbursmentPointDialog
+                isClosed={() => setIsDialogOpen(!isDialogOpen)}
```

Comment:
Je pense qu'il est plus sage de mettre false ici, quand on close le dialog on veut toujours le fermer. 
```suggestion
                isClosed={() => setIsDialogOpen(false)}
```